source/plugins/go/src/oms.go

```diff
@@ -161,6 +161,10 @@ var (
 	ContainerLogSchemaV2 bool
 	// container log schema version from config map
 	ContainerLogV2ConfigMap bool
+	// Kubernetes Metadata enabled through configmap flag
+	KubernetesMetadataConfigMap bool
```

Comment:
I think this should be renamed. Something like KubernetesMetadataEnabled?

Reply:
The reason why I put this name is because this feature might be expanded to enable through ux in future, let me know your think

Reply:
not sure I understand. can you expand?

Reply:
Like in future if we added the kuberneteMetadata enabled through portal/dcr, we will have two flags deciding whether  kuberneteMetadata  enabled is true or not

Reply:
i see. i think we should be fine with using KubernetesMetadataEnabled and can tackle UX case in the future if it happens considering we are planning on converging to one of the two options